Demo/Common/Networking/SessionRequest.swift

```diff
@@ -46,6 +47,8 @@ internal struct SessionRequest: APIRequest {
         //        try container.encode(installmentOptions, forKey: .installmentOptions)
 
         if ConfigurationConstants.current.cardComponentConfiguration.showsStorePaymentMethodField {
+            AdyenAssertion.assert(message: "API version should be greater than 69 to apply card component's store payment method field",
```

Comment:
👍  how about additionally adding a small info text to the the toggle in the settings?

Reply:
Maybe it would also be a good idea to set the default api version to 70 so it supports all the features of the demo app :)

Reply:
@goergisn I discussed this with Leo and he also said that we should keep the latest version  to make sure that everything will work and Also, because it is important that merchants will use the latest versions of our products;.